ministryofjustice/analytics-platform-control-panel
# DPAT-1711 Change the userinfo js for nomis apps

The PR is to address issue https://github.com/ministryofjustice/data-platform/issues/1711

Merging this PR will have the following side-effects:
- No

## :mag: What should the reviewer concentrate on?
- The java script

## :technologist: How should the reviewer test these changes?
The following approach only work if you have dev account on hmpps-auth dev server
- Login the auth0 alpha tenant
- find a hmpps-auth social connection ( I used `bold-case-info-dashboard-dev` ) 
- click it to see the connection's detail 
- use `try connection` to test it

## :books: Documentation status
